src/components/ProjectIdeaForm.tsx

```diff
@@ -101,10 +113,10 @@ const ProjectIdeaForm: React.FC<ProjectIdeaFormProps> = ({
 			}
 		};
 
-		if (user) {
-			void fetchProjectDetails();
-		}
-	}, [user]);
+		fetchProjectDetails().catch((err) =>
+			console.error("Unexpected error:", err),
+		);
+	}, [user, projectId]);
```

Comment:
Hi Fla,

Thank you for the thorough PR description detailing the steps for testing your new features. Additionally, you have adhered to our current component pattern when introducing new components.

For future enhancements, consider utilizing Next.js's prefetch method for navigation. Prefetching page data before redirecting can lead to faster navigation and an improved user experience.

Nonetheless, this has been really well done. Thank you again for the comprehensive PR description.
Kudos!

Reply:
I have tested this on the deployed preview and everything works as expected! Ready for merging.

Reply:
Thank you, Kyron! I appreciate the kudos :)  I'll take your suggestion for Next.js's prefetch method for next time. I'll go ahead and merge :)